packages/vector_graphics_compiler/lib/src/svg/numbers.dart

```diff
@@ -34,13 +32,8 @@ double radians(double degrees) => degrees * math.pi / 180;
 /// taking into account absolute and relative units
 /// (`px`, `em` or `ex`).
 ///
-/// Passing an `em` value will calculate the result
-/// relative to the provided [fontSize]:
-/// 1 em = 1 * `fontSize`.
-///
-/// Passing an `ex` value will calculate the result
-/// relative to the provided [xHeight]:
-/// 1 ex = 1 * `xHeight`.
+/// `rem`, `em`, and `ex` are currently parsed but
+/// sizing is deferred until runtime.
```

Comment:
I don't think we should do this.  We should just make them specify it at compile time. Otherwise we won't know about how to deal with overdraw or blending right?

Reply:
You won't be able to easily do this with text, unless you enforce that we not only know the font used ahead of time, but have access to the font file to be able to accurately measure glyphs.

Reply:
Does this mean we need to encode the unit scale though?

And same question for "currentColor"

Reply:
If we want to support it at runtime, definitely